nltk/metrics/distance.py

```diff
@@ -260,80 +260,90 @@ def jaro_winkler_similarity(s1, s2, p=0.1, max_l=4):
         American Statistical Association: 354-359.
     such that:
 
-        jaro_winkler_sim = jaro_sim + ( l * p * (1 - jaro_sim) )
+        jaro_winkler_sim = jaro_sim + ( l_ * p * (1 - jaro_sim) )
```

Comment:
I think it's cleaner to use `l` here to be clearer when communicating the formula.

Reply:
I think it's okay to use `l` in the code too instead of `l_` since the function is locally scoped, no global variable.